packages/edit-site/src/components/navigation-sidebar/navigation-panel/template-navigation-item.js

```diff
@@ -50,15 +50,19 @@ export default function TemplateNavigationItem( { item } ) {
 				onMouseEnter={ () => setIsPreviewVisible( true ) }
 				onMouseLeave={ () => setIsPreviewVisible( false ) }
 			>
-				<div className="edit-site-navigation-panel__template-item-title">
-					{ 'draft' === item.status && <em>{ __( '[Draft]' ) }</em> }
-					{ title }
-				</div>
-				{ description && (
-					<div className="edit-site-navigation-panel__template-item-description">
-						{ description }
+				<span className="components-menu-item__info-wrapper">
```

Comment:
It seems this classname is specific for the `MenuItem` component.  I assume this is being used for style reasons?

If it doesn't make sense to use the MenuItem here maybe we should either:
* Update the NavigationItem (I think that is the parent being used here) styles.  Although this would effect other uses of that component so Im not sure if that is desired.
* Adjust the style rules under `.edit-site-navigation-panel__template-item` to achieve the desired effect.